source/computer/linux101.rst

```diff
@@ -1,11 +1,61 @@
 Linux 入门
 ==========
 
-本章主要介绍 Linux 系统的入门级基础。
-
 .. toctree::
    :maxdepth: 2
+   :hidden:
 
    command-line 
    file-operations
    environment-variable
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-03-02
+:预计花费时间: 10 分钟
+
+----
+
+本章主要介绍 Linux 系统的入门基础。熟悉和掌握本章内容后，基本就可以在 Linux/macOS 系统上
+开展日常研究工作了。若想深入了解 Linux，可以参考一些较系统的资料进行学习。工作中若遇到问题，
+可以随时查阅。推荐通过以下方式或资料进行查阅或学习 Linux：
+
+1.  使用 Google 搜索
+2.  Linux 101
+
+    - 主页：https://101.lug.ustc.edu.cn/
+    - 作者：中国科学技术大学 Linux 用户协会
+    - 内容：系统地介绍 Linux 基础
+
+3.  Bash 脚本教程
+
+    - 主页：目前存在两个网站，尚不确定哪个是未来主要维护版本
+
+      - https://www.bookstack.cn/books/bash-tutorial：书栈网版本，内容似乎更全
+      - https://wangdoc.com/bash/：网道网站，正在建设中
```

Comment:
This is the repository for the wangdoc site. I think this one is well maintained.

Reply:
?? you mean only keep the second one? but it is still under development, and some content of the first doc have not been migrated to wangdoc.

Reply:
Updated in 9a3ef2e.